### PR TITLE
DO-436: Wish list style.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/member/MembersOccasion/MembersOccaisonGrid/MemberOccasionCard/MemberOccasionCard.jsx [hunk 1]
```diff
@@ -80,18 +83,40 @@ class MemberOccasionCard extends React.Component {
...
+              <Container className="member-occasion-today__content">
+                <MemberOccasionCardName memberId={id} firstName={firstName} lastName={lastName} />
+                <MemberOccasionCardDate date={date} timePeriod={timePeriod} />
+                {this.renderMessageBox(id, parentId)}
+                {this.renderAlreadyWishedContent()}
+              </Container>
+            </Container>
+          </Container>
+        );
+      }
```
Comment (ayushmalik2110) [inline src/components/member/MembersOccasion/MembersOccaisonGrid/MemberOccasionCard/MemberOccasionCard.jsx:103]: You don't have to apply additional check for today as we already applied it on renderMessageBox() and renderAlreadyWishedContent()
image
image

Comment (amitjain24524) [reply src/components/member/MembersOccasion/MembersOccaisonGrid/MemberOccasionCard/MemberOccasionCard.jsx:103]: Two different Container style for today and week, month birthday, So we need to add a condition for this.

Comment (ayushmalik2110) [reply src/components/member/MembersOccasion/MembersOccaisonGrid/MemberOccasionCard/MemberOccasionCard.jsx:103]: Okay.

File: src/components/routes/Member/AuthenticatedMember/MemberWishesRoute/MemberWishesRoute.jsx [hunk 2]
```diff
@@ -83,18 +131,22 @@ class MemberWishesRoute extends React.Component {
...
+        <MemberWishesMusic isAnniversary={this.state.anniversaryType} refFunc={(ref) => { this.audioEl = ref; }} />
         <RouteHeader
           title={routeHeaderText}
           showBackButton
-          backButtonRoute={backButtonRoute}
+          backButtonRoute={Routes.MEMBER_DASHBOARD}
         />
-        <MemberWishesRouteContent messageArray={messageArray} />
+        <Container className="member-wishes-container__play-icon">
+          <button onClick={this.togglePlay}>{this.state.play ? <IconSoundOn />
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberWishesRoute/MemberWishesRoute.jsx:143]: Use Button component

File: src/components/routes/Member/AuthenticatedMember/MemberWishesRoute/MemberWishesRoute.jsx [hunk 3]
```diff
@@ -56,6 +66,27 @@ MemberWishesRouteContent.defaultProps = {
   messageArray: [],
 };
 
+
+const MemberWishesMusic = (props) => {
+  if (props.isAnniversary === true) {
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberWishesRoute/MemberWishesRoute.jsx:71]:
```suggestion
  if (props.isAnniversary) {
```

File: src/components/routes/Member/AuthenticatedMember/MemberWishesRoute/MemberWishesRoute.jsx (hunk 2 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberWishesRoute/MemberWishesRoute.jsx]: done

File: src/components/routes/Member/AuthenticatedMember/MemberWishesRoute/MemberWishesRoute.jsx (hunk 3 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberWishesRoute/MemberWishesRoute.jsx]: done

File: src/components/member/MembersOccasion/MembersOccaisonGrid/MemberOccasionCard/MemberOccasionCard.jsx [hunk 4]
```diff
@@ -80,18 +83,40 @@ class MemberOccasionCard extends React.Component {
...
-        <Container className="member-occasion-card-container__content">
-          <Link to={getMemberDetailsRouteLink()}>
-            <MemberOccasionCardImage imageUrl={imageUrl} />
-            <MemberOccasionCardName memberId={id} firstName={firstName} lastName={lastName} />
-            <MemberOccasionCardDate date={date} timePeriod={timePeriod} />
-          </Link>
-          {this.renderMessageBox(id, parentId)}
-          {this.renderAlreadyWishedContent()}
+    const MemberOccasionCardDetail = () => {
+      if (selectedCard === 'Today') {
```
[the remark this answers is not in the document]
Comment (amitjain24524) [reply src/components/member/MembersOccasion/MembersOccaisonGrid/MemberOccasionCard/MemberOccasionCard.jsx]: done

**Pull request merged**